src/utils/DockerUtil.js

```diff
@@ -159,6 +160,12 @@ var DockerUtil = {
         containerData.HostConfig.PublishAllPorts = true;
       }
 
+      containerData.NetworkingConfig = {
+        EndpointsConfig: _.mapObject(containerData.NetworkSettings.Networks, (network) => {
```

Comment:
Unfortunately this doesn't always work - If no-network has been set, you're creating the container for the first time, you'll end up with:
`TypeError: Cannot read property 'Networks' of undefined`